src/v/storage/log_manager.cc

```diff
@@ -284,6 +284,14 @@ log_manager::get(const model::topic_namespace& tn) {
     return r;
 }
 
+absl::flat_hash_set<model::ntp> log_manager::get() const {
+    absl::flat_hash_set<model::ntp> r;
```

Comment:
i don't think this code benefits from the cacheline optimization of flat_hash_set as the NTP is at least 52bytes. 

flat* are really designed around lookups (int to pointer) or int-to-int 16byte cache line optimizations AFAIK. but let me re-read the original design before changing.